pkg/apis/v1beta1/aksnodeclass_status.go

```diff
@@ -0,0 +1,104 @@
+/*
+Portions Copyright (c) Microsoft Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"fmt"
+
+	"github.com/awslabs/operatorpkg/status"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	ConditionTypeImagesReady            = "ImagesReady"
+	ConditionTypeKubernetesVersionReady = "KubernetesVersionReady"
+)
+
```

Comment:
Wondering if in all of these files we want a big WARNING: YOU MUST KEEP THIS AND v1alpha1 IN SYNC

Or something? We could actually add some automation for it, but not sure if worth.